internal/common/start_processing.go

```
@@ -7,132 +7,128 @@ import (
-	var payload models.Payload
+	var flow bmodels.Flows
+	var payload bmodels.Payloads
 	for _, step := range initialSteps {
-		if step.Pending {
-			data, _, canceled, no_pattern_match, failed, err := processStep(flow, payload, initialSteps, step, execution)
+		if step.Status == "pending" {
+			data, success, err := processStep(cfg, actions, loadedPlugins, flow, payload, initialSteps, step, execution)
 			if err != nil {
-				executions.EndWithError(execution)
+				// cancel remaining steps
+				internal_executions.CancelRemainingSteps(cfg, execution.ID.String())
```

golangci-lint (1.62.2)

Error return value of `internal_executions.CancelRemainingSteps` is not checked

(errcheck)